torchprime/torch_xla_models/configs/model/llama.yaml

```diff
@@ -0,0 +1,17 @@
+vocab_size: 128256
```

Comment:
Does it make sense to name this file `llama-3-8b.yaml`, since this describes the exact model?